src/edit/LockHandle.js

```diff
@@ -12,6 +12,59 @@ L.LockHandle = L.EditHandle.extend({
 	_onHandleDrag: function() {
 	},
 
+  _bindListeners: function() {
+    this.on(
+      {
+        dragstart: this._onHandleDragStart,
+        drag: this._onHandleDrag,
+        dragend: this._onHandleDragEnd,
+        mouseover: this._setTooltip
+      },
+      this
+    );
+
+    this._handled._map.on("zoomend", this.updateHandle, this);
+
+    this._handled.on("update", this.updateHandle, this);
+  },
+
+  _setTooltip: function () {
+    
+	  var Icon = L.Icon.extend({
+      options: {
+        iconSize: [ 50, 50 ],
+        iconAnchor: [ 25, 50 ]
+      }
+    });
+   
+	  var question = new Icon({
+      iconUrl: '../assets/icons/markers/question-mark.png'
+    });
+	  
+	  this._questionMark = new L.marker(this._handled.getCorner(this._corner), { icon: question })
+		  .bindTooltip("Locked!")
```

Comment:
could we show a tooltip over the X handles that says Locked or Image is locked?

Done here.